xml/bp_troubleshooting_tips.xml

```diff
@@ -645,13 +645,7 @@ Succeeded: 20 addresses from 10 minions average rtt 14.16 ms
 <screen>&prompt.cephuser;ceph device ls</screen>
 
  </sect1>
- <sect1 xml:id="multisite-replication-config">
-  <title>Multi-site Replication Configurations</title>
```

Comment:
I'm not sure why we had this in there, @tbazant do you have any insight? This isn't a section we previously had in any guide.